BookingsApi/Testing.Common/Builders/Domain/RefDataBuilder.cs

```diff
@@ -6,21 +7,57 @@ namespace Testing.Common.Builders.Domain
     public class RefDataBuilder
     {
         public List<HearingVenue> HearingVenues { get; set; }
+        public List<JusticeUser> Users { get; set; }
 
 
         public RefDataBuilder()
         {
             InitHearingVenues();
+            InitUsers();
         }
         
         private void InitHearingVenues()
         {
-           HearingVenues = new List<HearingVenue>()
-           {
-               new HearingVenue(1, "Birmingham Civil and Family Justice Centre"),
-               new HearingVenue(2, "Manchester Civil and Family Justice Centre"),
-               new HearingVenue(3, "Taylor House Tribunal Hearing Centre"),
-           };
+            HearingVenues = new List<HearingVenue>()
+            {
+                new HearingVenue(1, "Birmingham Civil and Family Justice Centre"),
+                new HearingVenue(2, "Manchester Civil and Family Justice Centre"),
+                new HearingVenue(3, "Taylor House Tribunal Hearing Centre"),
+            };
+        }
+        
+        private void InitUsers()
+        {
+            Users = new List<JusticeUser>()
```

Comment:
we should assign roles to these users too

Reply:
this is a mock and don't need role in the used context